jobs/src/main/scala/dcos/metronome/scheduler/SchedulerConfig.scala

```diff
@@ -2,11 +2,13 @@ package dcos.metronome.scheduler
 
 import dcos.metronome.repository.impl.kv.ZkConfig
 import mesosphere.marathon.AllConf
+import mesosphere.marathon.core.task.termination.TaskKillConfig
 
 import scala.concurrent.duration.FiniteDuration
 
 trait SchedulerConfig extends ZkConfig {
   def scallopConf: AllConf
+  def taskKillConfig: TaskKillConfig
```

Comment:
We need the KillConfig but I'm not familiar with Scallop and the approach of having multiple config traits.   The metronome way was to have traits that are not extensions of scallop...  looking for advice here.

Reply:
taskKillConfig I think is derived from multiple scallop options. Does it compile? Does it work?